e2e/process-services-cloud/task-header-cloud.e2e.ts

```diff
@@ -149,4 +154,19 @@ describe('Task Header cloud component', () => {
         expect(taskHeaderCloudPage.getParentTaskId())
             .toEqual(subTask.entry.parentTaskId === null ? '' : subTask.entry.parentTaskId);
     });
+
+    it('[C309699] SShould have all the fields on the task header cloud readonly when the task is unclaimed', () => {
```

Comment:
Should we extend this test to check for all the other fields on the task header cloud as well? I realise that the bug was for these 2 fields, but will be nice to cover all the fields in the test.

Reply:
I think it's a very good idea, i will add all fields.